files/templates/alertmanager/notification-template.tmpl

```diff
@@ -1,13 +1,23 @@
 {{ define "__alertmanager" }}Alertmanager{{ end }}
-{{ define "__alertmanagerurl" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}&silenced=false&inhibited=false&active=true&filter=%7Balertname%3D%22{{ .CommonLabels.alertname }}%22%7D{{ end }}
+
+{{ define "__alertmanagerurl" }}
+[[- if .MimirEnabled ]]
+[[ .GrafanaAddress ]]/alerting/Mimir/{{ .CommonLabels.alertname }}/find
+[[- else ]]
+{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}&silenced=false&inhibited=false&active=true&filter=%7Balertname%3D%22{{ .CommonLabels.alertname }}%22%7D
+[[- end ]]
+{{ end }}
+
 {{ define "__dashboardurl" -}}{{ if match "^https://.+" (index .Alerts 0).Annotations.dashboard }}{{ (index .Alerts 0).Annotations.dashboard }}{{ else }}[[ .GrafanaAddress ]]/d/{{ (index .Alerts 0).Annotations.dashboard }}{{ end }}{{- end }}
 {{ define "__runbookurl" -}}https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/{{ (index .Alerts 0).Annotations.opsrecipe }}{{- end }}
 
+{{ define "__queryurl" }}
 [[- if .MimirEnabled ]]
-{{ define "__queryurl" }}[[ .GrafanaAddress ]]/alerting/Mimir/{{ .CommonLabels.alertname }}/find{{end}}
+[[ .GrafanaAddress ]]/alerting/Mimir/{{ .CommonLabels.alertname }}/find
```

Comment:
Does that work?

Reply:
I was afraid of the formatting

Reply:
Well if you generate the tests using go test ./... -update then we will see :D

Reply:
Please don't merge master in my PR, because it creates conflicts with my changes as I didn't know you merged on your side...